Resprited fire lockers

Moomoobeef opened the pull request:
## About the PR
I was irritated by how fire lockers and nitrogen lockers are not very visually distinct from eachother, so I took the opportunity to do some respriting. Fire lockers now have bright yellow stripes on them. At first I removed the white stripe from nitrogen lockers but I decided to add it back, and I decided to (for consistency) also add a white stripe to the oxygen locker since it was the only one of the set without one to begin with.

## Why / Balance
I like it better this way

## Technical details
I wanted to use the same trick as in #37869 to make fire lockers stripes glow dimly, but I am not sure how (or if it's possible) to do this with EntityStorageVisuals. I figure there's probably a way to do it and I just don't know how :P 
So please do tell.

## Media
![lockers_shaded](https://github.com/user-attachments/assets/4102d89f-2d55-4ef4-89ea-87fc4493be7c)
![lockers_unshaded](https://github.com/user-attachments/assets/64974343-0bfd-48d6-9ae5-ff8b033c74d1)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
Shouldn't be any

**Changelog**
Nah

Moomoobeef committed:
initial commit

changed sprites for existing n2 lockers and fire lockers to be more distinct from eachother. Was intending to make fire-lockers have glowing stripes the same way the high-vis does but couldn't figure out how to make EntityStorageVisuals do what I need.

Moomoobeef committed:
added the stripe back to n2 wall cabinet

So I removed the stripe from the N2 cabinets because the O2 locker hs no stripe, and to better distinguish it from the fire locker. But then I found out that the O2 *wall cabinet* does have a stripe. I decided to add the stripe back to the wall N2 cabinet because of this, and because it's already dinstinct enough from the fire wall cabinet because the fire wall cabinet is a different shape (why?)

Moomoobeef committed:
modifications again

made fire cabinets have only yellow stripes following feedback from EMO, and then decided to add white stripes back to N2 lockers, and to (for consistency) add a stripe to the O2 locker

Moomoobeef committed:
updated fire_door_glow

Moomoobeef committed:
Merge branch 'master' into fire-lockers-resprite

Moomoobeef committed:
this is the part where I fix it with a dumb fucking text editor that will do WHAT I TELL IT TO DO and NOTHING ELSE

SlimmSlamm commented:
Shouldn’t we keep the N2 lockers for Slimes and Voxes? Plus whatever other race in the future uses N2

Moomoobeef commented:
> Can you rotate the anchorable fire locker's fire so that it looks like the wall locker's or vice versa?

That is outside the scope of this PR
> Also I think you put two "after" images instead of a "before" one lmao. Sick work though!

Both are after, I was showing them off both with and without shaded lighting

Moomoobeef committed:
made locker stripes consistent

when I changed my mind on removing the stripe from the N2 locker I forgot to add it back to the other sprites. I also forgot to add the stripe I added to the O2 locker to its other sprites. This change does that.

Moomoobeef committed:
update the meta.json

Warpzoned commented:
> > Can you rotate the anchorable fire locker's fire so that it looks like the wall locker's or vice versa?
> 
> That is outside the scope of this PR

I was talking about the sprite's fire itself 😭, on one of them its "tail" is to the left and on the other its to the right, and having it rotated/flipped would be good for consistency I think

> > Also I think you put two "after" images instead of a "before" one lmao. Sick work though!
> 
> Both are after, I was showing them off both with and without shaded lighting

Gotcha!

Moomoobeef commented:
> I was talking about the sprite's fire itself 😭, on one of them its "tail" is to the left and on the other its to the right, and having it rotated/flipped would be good for consistency I think

Ah, I could do that I suppose. It hadn't occurred to me (the one on the wall locker was already like that before I started)

ps3moira commented:
The yellow band should be moved around it's placement is a little strange

Moomoobeef commented:
> The yellow band should be moved around it's placement is a little strange

on the wall locker, the standing locker, or both?

Moomoobeef committed:
made the fire locker sprites more similar to the n2/o2 locker sprites

wait does that defeat the purpose of this PR? uhhhhhhhhhhhhhhhhhhhh... No it's fine because the stripe is bright yellow.

Moomoobeef commented:
![image](https://github.com/user-attachments/assets/299cbdf9-04c8-4882-9bca-cf43e4b53f06)
so I liked how with the two-toned band it looked reflective, however ingame this actually looks a bit shit

Moomoobeef commented:
sooooo I think I am going to change it, I'm going to try and fix it to be less ugly because I still do like the two-toned stripe, but failing that I'll just copy the other stripes and make it yellow

Moomoobeef commented:
![image](https://github.com/user-attachments/assets/e40c2bb3-e498-42fd-af4a-4de3689fe8e7)

Moomoobeef committed:
the last commit was a bitch ass motherfucker

it looked bad, so I made it betterer

Moomoobeef commented:
![image](https://github.com/user-attachments/assets/ec6d489f-c9de-41ec-a2fe-3ff395397102)
I honestly think it looks ugly with the wall locker fire on it, it's just too big
